src/game.cpp

```diff
@@ -12,23 +13,26 @@ Game::Game(const GameSettings& settings) : settings_(settings) {
                    sf::Style::Close | sf::Style::Titlebar | sf::Style::Resize,
                    window_settings);
 
+    // open menu screen
+    open_menu();
```

Comment:
ten `game.cpp` to jest totalny syf, będziemy musieli to przepisać i zmodularyzować jakoś bo ugrzęźniemy w tym bagnie

ale na razie git